payloads/v10/channel.ts

```diff
@@ -2413,13 +2429,24 @@ export type APIComponentInActionRow = APIComponentInMessageActionRow | APICompon
 /**
  * @see {@link https://discord.com/developers/docs/components/reference#action-row}
  */
-export type APIComponentInMessageActionRow = APIButtonComponent | APISelectMenuComponent;
+export type APIComponentInMessageActionRow = APIButtonComponent | APIStringSelectComponent;
+
+/**
+ * @see {@link https://discord.com/developers/docs/components/reference#action-row}
+ */
+export type APIComponentInModal = APIComponentInLabel | APIComponentInModalActionRow;
```

Comment:
Shouldn't this be APILabelComponent | APIModalActionRowThing

Reply:
`APIModalComponent` above seems to be that. The naming conventions don't really seem intuitive, but it is what it is.

Reply:
Well no, APIModalComponent is anything in a modal, ComponentInX is supposed to be direct components of X... So ActionRow or Label